### DISCUSSION TITLE
please, tell me, this library is still working?

### DISCUSSION THREAD

**Discussion by MelodyEars** [kaliiiiiiiiii/Selenium-Profiles#59] [Q&A] [answered]
hello  @kaliiiiiiiiii, please tell me this library is working?
how can i replace the gpu am i doing it right?
and what can I add to avoid detection on this site
'https://abrahamjuliot.github.io/creepjs/'
any suggestions please

```python
PROFILE = {
    "Windows": {
        "options": {
            "gpu": 'ANGLE (Intel, Intel(R) HD Graphics 520 Direct3D11 vs_5_0 ps_5_0)',
            "window_size": {
                "x": 1366,
                "y": 768}
        },
        "cdp": {
            "touch": True,
            "maxtouchpoints": 10,
            "cores": 4,
            "patch_version": True,
            "emulation": {"mobile": False, "width": 1366, "height": 768, "deviceScaleFactor": 1,
                          "screenOrientation": {"type": "landscapePrimary", "angle": 0}},
            "useragent": {
                "platform": "Win32",
                "acceptLanguage": "en-US",
                "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36",
                "userAgentMetadata": {
                    "brands": [{"brand": "Google Chrome", "version": "114"},
                               {"brand": "Chromium", "version": "114"},
                               {"brand": "Not=A?Brand", "version": "24"}],
                    "fullVersionList": [{"brand": "Google Chrome", "version": "114.0.5735.134"},
                                        {"brand": "Chromium", "version": "114.0.5735.134"},
                                        {"brand": "Not=A?Brand", "version": "24.0.0.0"}],
                    "fullVersion": "114.0.5735.134",
                    "platform": "Windows",
                    "platformVersion": "10.0.19045",
                    "architecture": "x86",
                    "model": "",
                    "mobile": False,
                    "bitness": "64",
                    "wow64": False}
            }
        }
    }
}
```

my code 

```python
from selenium_profiles import webdriver

mydriver = webdriver.Chrome(PROFILE["Windows"], uc_driver=False)  # or pass seleniumwire-options
driver = mydriver.start()

# get url
try:
    driver.get('https://abrahamjuliot.github.io/creepjs/')  # test fingerprint

    input("Press ENTER to exit: ")
finally:
    driver.quit()  # Execute on the End!
    
```

**Comment by ttlns** [comment]
No, emulating a different GPT-vendor isn't supported for now. 
@ArtDanger What are you getting detected on?

**Reply by MelodyEars** [reply]
Unfortunately, my accounts are banned due to the video card. Why can anti detect dolphin normally change gpu, RAM and CPU (number of cores), but here it is not possible?
I used undetected_chromedriver, before that I thought I was completely protected from detection, but reddit sees the components of my computer and bans me.

**Comment by ttlns** [comment]
@ArtDanger are you referring to https://anty.dolphin.ru.com/ ?
You can try playing with setting `profile["options"]["gpu"]` to `True` or `False`

**Reply by MelodyEars** [reply]
1. yes
2. Oh, I thought that I can transfer the parameter of the video card that I want to use in the driver there

I want to thank you for your response

**Reply by MelodyEars** [reply]
I'll tell you like this, a friend of mine was banned from her reddit account, and no matter how much she tried to create new accounts, they were all banned (with a proxy too). she downloaded this browser and without a proxy her accounts are not banned

it looks like I'm advertising everyone to use it, hahaha )))

**Comment by ttlns** [comment, marked as answer]
> 1. yes 2 Oh, I thought that I can transfer the parameter of the video card that I want to use in the driver there

Yeah, that would be possible with overwriting those values using javascript. The problem with that is that overwriting currently is detectable. 

For more information, you might have a look at https://github.com/abrahamjuliot/creepjs/issues/147 or [adtechmadness.wordpress.com/2019/03/23/javascript-tampering-detection-and-stealth](https://adtechmadness.wordpress.com/2019/03/23/javascript-tampering-detection-and-stealth/)
